pkg/disk/cloud.go

```diff
@@ -521,6 +521,24 @@ func getDisk(diskID string, ecsClient *ecs.Client) []ecs.Disk {
 	return diskResponse.Disks.Disk
 }
 
+func tagDiskUserTags(diskID string, ecsClient *ecs.Client, tags map[string]string) {
+	addTagsReq := ecs.CreateAddTagsRequest()
+	var userTags []ecs.AddTagsTag
+	for k, v := range tags {
+		userTags = append(userTags, ecs.AddTagsTag{Key: k, Value: v})
+	}
+	addTagsReq.Tag = &userTags
+	addTagsReq.ResourceType = "disk"
+	addTagsReq.ResourceId = diskID
+	addTagsReq.RegionId = GlobalConfigVar.Region
+	_, err := ecsClient.AddTags(addTagsReq)
+	if err != nil {
+		log.Warnf("tagDiskUserTags: AddTags error: %s, %s", diskID, err.Error())
+		return
```

Comment:
This error is otherwise ignored. Is this intentional? Should the failure block PV creating?

Reply:
Normally, a failed tag shouldn't impact the PV. I'm just referencing when `tagDiskAsK8sAttached` failed while attaching, it's ignored too. @mowangdk

Reply:
yep, failed tag has no impact to PV. we should keep the original logic